src/Components/NavBar/NavBarHome.js

```diff
@@ -88,9 +119,9 @@ const ButtonWrapper = styled.div`
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 54px;
+    width: 60px;
     height: 17px;
-    padding: 13px 12px 8px 12px;
+    padding: 11px 12px 11px 12px;
```

Comment:
```suggestion
    padding: 11px 12px;
```
로 조금 더 깔끔하게 정리 가능할 것 같습니다~!

Reply:
이건 종민님 코드이긴 한데 제가 수정해 놓을게요~ ㅎㅎ 감사합니다~